Packs/Automox/README.md

```diff
@@ -1 +1,16 @@
 # Automox Content Pack for XSOAR
+The Automox Content Pack for XSOAR empowers users by providing the resources to remediate vulnerabilities and administrate your IT organization with ease. Use the Automox integration to create device groups, update devices, and run policies. This content pack also includes a sub-playbook to use in your vulnerability remediation workflows to identify vulnerable devices and automatically generate the remediation tasks necessary to patch them in Automox.
+
+# What does this pack do?
```

Comment:
Personal recommendation would be to organize this from general management first down to vuln capabilities

Reply:
The first iteration actually sounded more like what you are describing lol. I opted for putting vulnerability remediation first, probably because it's a hot topic atm. I see your point and will reorganize this paragraph.

Reply:
Added a suggestion: https://github.com/ax-jmiller/content/pull/3/files#r833684664